src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingSolutionCrawlerLogger.cs

```diff
@@ -219,10 +219,12 @@ public static void LogIncrementalAnalyzerProcessorStatistics(int correlationId,
                 }
             }));
 
+#if false // Not used in unit testing crawling
             foreach (var analyzer in analyzers)
             {
                 analyzer.LogAnalyzerCountSummary();
             }
+#endif
```

Comment:
an example of working backwards.  becausse unit testing doesn't use LogAnalyzerCountSummary, we ifdefed it out.  so this loop here is also unnecessary.